WorldWideWeb/NXShims.h

```diff
@@ -0,0 +1,85 @@
+//
+//  NXShims.h
+//  WorldWideWeb
+//
+//  Created by Chris Brandow on 8/23/24.
+//
+
+#ifndef NXShims_h
+#define NXShims_h
+
+
+#endif /* NXShims_h */
+
+/// These are largely just copied from Next Manual
+/// http://www.bitsavers.org/pdf/next/Release_1_Dec90/NEXTstep_Reference_Volume_1_Dec90.pdf
+/// http://www.bitsavers.org/pdf/next/Release_1_Dec90/NEXTstep_Reference_Volume_2_Dec90.pdf
+
+typedef struct NXRect {
+    NXPoint origin;
+    NXSize size;
+} NXRect;
+
+
+typedef struct NXTextBlock {
```

Comment:
I think there's a typo here:

```suggestion
typedef struct _NXTextBlock {
```